pom.xml

```diff
@@ -489,14 +505,34 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>1.${java.target.version}</source>
+                    <target>1.${java.target.version}</target>
```

Comment:
in order to be able to upgrade to junit5, is it a good idea to use different jdk for unit testing and code compilation (as described here -> https://gist.github.com/aslakknutsen/9648594)?